TwoPaneLayout/README.md

```diff
@@ -73,7 +128,7 @@ About some common use case for the two panes, please check out [user interface p
     }
     ```
 
-4. Build layout with **TwoPaneLayout**. Please refer to the [sample](https://github.com/microsoft/surface-duo-compose-sdk/tree/main/TwoPaneLayout/sample) for more details.
+4. Build layout with **TwoPaneLayout** or **TwoPaneLayoutNav**. Please refer to the [sample](https://github.com/microsoft/surface-duo-compose-sdk/tree/main/TwoPaneLayout/sample) and [nav sample](https://github.com/microsoft/surface-duo-compose-sdk/tree/main/TwoPaneLayout/nav_sample) for more details.
```

Comment:
Can we please add some screenshots from TwoPaneLayoutNav sample?

Reply:
I added in a GIF, let me know if you think there are any other spots that could use images!